### PR TITLE
Bundle the ipxe template in the osbuilder

### PR DESCRIPTION
**Pull request by Itxaka**
So it can be used when building ipxe stuff instead of being walled in the kairos repo

### CODE REVIEW

**Comment by jimmykarily** [comment]
was this template in the kairos repo? I can't find it.

**Comment by Itxaka** [comment]
> was this template in the kairos repo? I can't find it.

not as a template, but as a script that produced the output with this template: https://github.com/kairos-io/kairos/blob/master/scripts/netboot.sh

File: tools-image/ipxe.tmpl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!ipxe
+
+set dns 8.8.8.8
+ifconf
+kernel ${RELEASE_URL}/${VERSION}/${ISO_NAME}-kernel root=live:${RELEASE_URL}/${VERSION}/${ISO_NAME}.squashfs initrd=${ISO_NAME}-initrd rd.neednet=1 ip=dhcp rd.cos.disable netboot nodepair.enable config_url=${config} console=tty1 console=ttyS0 rd.live.overlay.overlayfs ${cmdline}
```
Comment (mauromorales) [inline tools-image/ipxe.tmpl:5]: how is the ISO_NAME populated? just to make sure we do it with the new names

**Review by mauromorales — APPROVED** [review]

File: tools-image/ipxe.tmpl (hunk 1 above)
Comment (mauromorales) [reply tools-image/ipxe.tmpl:5]: found it https://github.com/kairos-io/kairos/pull/1735/files

**Comment by jimmykarily** [comment]
> > was this template in the kairos repo? I can't find it.
> 
> not as a template, but as a script that produced the output with this template: https://github.com/kairos-io/kairos/blob/master/scripts/netboot.sh

so we can remove that script after this is merged (and consumed) right?

**Comment by Itxaka** [comment]
> > > was this template in the kairos repo? I can't find it.
> > 
> > 
> > not as a template, but as a script that produced the output with this template: [kairos-io/kairos@`master`/scripts/netboot.sh](https://github.com/kairos-io/kairos/blob/master/scripts/netboot.sh?rgh-link-date=2023-08-18T07%3A57%3A32Z)
> 
> so we can remove that script after this is merged (and consumed) right?

yup, already a PR in there to do that :D

**Pull request merged**